.ci_config

```diff
@@ -0,0 +1 @@
+Subproject commit 31f2b420a0816e459273078445dacd805d9501e3
```

Comment:
The "new" or "preferred" way of utilizing travis is to have travis clone the industrial_ci repo, instead of including it as a submodule.  This ensures that the latest version of industrial_ci is always used.  The sub-module approach still works though.